src/core/numerics/ion_updater/ion_updater.h

```diff
@@ -80,7 +80,7 @@ void IonUpdater<Ions, Electromag, GridLayout>::updatePopulations(Ions& ions, Ele
     resetMoments(ions);
     pusher_->setMeshAndTimeStep(layout.meshSize(), dt);
 
-    if (mode == UpdaterMode::moments_only)
+    if (mode == UpdaterMode::domain_only)
```

Comment:
I'm putting the comment here although it is for the line below.
Maybe  `updateMomentsOnly_` now needs to be renamed `updateAndDepositDomain()` and `updateAll` may become `updateAndDepositAll_` ? if you have better suggestions maybe?